Automattic/pocket-casts-ios
# Pride: add new default icon

Adds a new amazing icon to celebrate the Pride Month.

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/f432719a-8636-4b74-bb0a-a4378f1ff326" width="300">

## To test

1. Install `release/7.39` in two different devices or simulator
1. In one of them, change the default icon
1. Checkout this branch
1. Run the app on the device you didn't changed the icon
1. ✅ The new pride icon should be displayed
1. Run the app on the device/simulator you changed the icon
1. ✅ The pre-selected custom icon should be displayed
1. Go to Settings > Appearance
1. ✅ The Pride icon should be displayed after "Red Velvet" with the "Pride 2023" label
1. Tap on it
1. ✅ The icon should change

Make sure to test these steps on iPhone and iPad.

## Instructios for 7.41

We'll revert back to the original icon in 7.41. In order to do so all that needs to be done is:

```
git revert a97888a5753076dd079e8ab1d786f7f2453bf50b
git revert 66fde22bb449d69f934dfc80cb07a454b55c0686
git revert 4bbba40e218aa18cf516212c69e53d8453a91549
git revert 09b1b6d736cc8ba9fb9c5d56091536a46d902d4d
```

We'll keep the icon there in Settings though! 🏳️‍🌈

## Checklist
